### ISSUE TITLE
WebAuthn Flows Broken in login.v2

### ISSUE THREAD

**Issue by ThoreKr** [keycloak/keycloak#32578]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

login/ui

### Describe the bug

As a follow-up to #32365, this issue also affects the login.v2 theme making it impossible to login using a security or registering a new one. The error in the js console is the same 

```
TypeError: Failed to resolve module specifier "rfc4648". Relative references must start with either "/", "./", or "../".
```

### Version

25.0.4

### Regression

- [X] The issue is a regression

### Expected behavior

Login with a security key or registration of a new one is possible.

### Actual behavior

The error mentioned above is thrown.

### How to Reproduce?

Have a keycloak instance with the login2 feature flag enabled.
Enable it in a realm and create a webauthn enabled browser flow.
Try to register a new key.

### Anything else?

It used to work in 24.0.x

**Comment by jonkoops** [comment]
Fix for this is up under #32589

**Comment by ahus1** [comment]
Closing as this has now been merged in the release 25 branch.
